language/snippets/src/main/java/com/example/language/Analyze.java

```diff
@@ -16,27 +16,24 @@
 
 package com.example.language;
 
-import com.google.cloud.language.v1.AnalyzeEntitiesRequest;
-import com.google.cloud.language.v1.AnalyzeEntitiesResponse;
 import com.google.cloud.language.v1.AnalyzeEntitySentimentRequest;
 import com.google.cloud.language.v1.AnalyzeEntitySentimentResponse;
```

Comment:
nit: what is a reason to leave v1 Request / Response imports?

Reply:
The `AnalyzeEntitySentiment` API is still V1. We need to keep it around until V2 is available.